src/controllers/tasks/statement.of.capital.controller.ts

```diff
@@ -3,13 +3,20 @@ import { RADIO_BUTTON_VALUE, STATEMENT_OF_CAPITAL_ERROR } from "../../utils/cons
 import { STATEMENT_OF_CAPITAL_PATH, TASK_LIST_PATH, urlParams } from "../../types/page.urls";
 import { Templates } from "../../types/template.paths";
 import { urlUtils } from "../../utils/url";
+import { getStatementOfCapitalData } from "../../services/statement.of.capital.service";
+import { Session } from "@companieshouse/node-session-handler";
+import { StatementOfCapital } from "private-api-sdk-node/dist/services/confirmation-statement";
 
-export const get = (req: Request, res: Response, next: NextFunction) => {
+export const get = async(req: Request, res: Response, next: NextFunction) => {
   try {
     const companyNumber = getCompanyNumber(req);
+    const session: Session = req.session as Session;
+    const transactionId: string = "";
```

Comment:
I appreciate that we already need to do a large refactor once the web journey actually creates a transaction.
but presumably the api call following this will fallover without a transaction id. So is there any need to get this pr in prior to the transaction id actually being present?

Reply:
or on second thoughts, why does the statement of capital endpoint even need a transaction id?